ci-operator/config/openshift/origin/openshift-origin-master.yaml

```diff
@@ -182,6 +186,38 @@ tests:
     env:
       TEST_SUITE: openshift/conformance/parallel
     workflow: openshift-e2e-vsphere-upi
+- artifact_dir: /tmp/artifacts
+  as: e2e-metal-ipi
+  steps:
+    cluster_profile: packet
+    env:
+      DEVSCRIPTS_CONFIG: |
+        IP_STACK=v4
+        NETWORK_TYPE=OpenShiftSDN
+    workflow: baremetalds-e2e
+- artifact_dir: /tmp/artifacts
+  as: e2e-metal-ipi-ovn-ipv6
+  steps:
+    cluster_profile: packet
+    env:
+      DEVSCRIPTS_CONFIG: |
+        IP_STACK=v6
+        NETWORK_TYPE=OVNKubernetes
+    workflow: baremetalds-e2e
+- artifact_dir: /tmp/artifacts
+  as: e2e-metal-ipi-ovn-dualstack
+  steps:
+    cluster_profile: packet
+    env:
+      DEVSCRIPTS_CONFIG: |
+        IP_STACK=v4v6
+        NETWORK_TYPE=OVNKubernetes
+    workflow: baremetalds-e2e
+- artifact_dir: /tmp/artifacts
+  as: e2e-metal-ipi-virtualmedia
+  steps:
+    cluster_profile: packet
+    workflow: baremetalds-e2e-virtualmedia
```

Comment:
these should be optional to start with, until proven to be stable.

Reply:
It's a good point. Usually in the other repos where we added those tests they are all option but `e2e-metal-ipi-ovn-ipv6` (note: some of these are also gating the release), I could follow the same approach?

Reply:
e2e-metal-ipi (IPv4), e2e-metal-ipi-dualstack and e2e-metal-ipi-virtualmedia marked as optional.